docs/src/sharing.rst

```diff
@@ -0,0 +1,183 @@
+
+.. code-block:: ts
+		
+    const myvalue = new SSMParameterProvider(this).getString("my-awesome-value");
+
+See the :doc:`context` topic for information about the :py:class:`SSMParameterProvider <@aws-cdk/cdk.SSMParameterProvider>`.
+
+.. _sharing_between_stacks:
+
+Sharing Information Between Stacks
+==================================
+
+You can share information between two stacks in the same app
+by using the **export** method on one stack and the **import** method on the other stack.
+
+First add a property to the class that defines the stack you are exporting from.
+The following example shows a stack with the property **myBucketRefProps**.
+
+.. code-block:: ts
+
+    class HelloCdkStack extends cdk.Stack {
+        public readonly myBucketRefProps: s3.BucketRefProps;
+
+	constructor(parent: cdk.App, name: string, props?: cdk.StackProps) {
+            super(parent, name, props);
+
+Next create a bucket and export it's *BucketRefProps* to the **myBucketRefProps** property.
+
+.. code-block:: ts
+
+    const mybucket = new s3.Bucket(this, "MyFirstBucket");
+
+    this.myBucketRefProps = mybucket.export();
+
+Create an interface, with one property, an **BucketRefProps** object.
+We'll use this interface to pass the reference to the bucket,
+as a set of bucket properties,
+to another stack.
+
+.. code-block:: ts
+
+    interface XferBucketProps {
+        theBucketRefProps: s3.BucketRefProps;
+    }
+
+Now create the class for the other stack.
+
+.. code-block:: ts
+
+    class MyCdkStack extends cdk.Stack {
+        constructor(parent: cdk.App, name: string, props: XferBucketProps) {
+            super(parent, name);
+
+            const myOtherBucket = s3.Bucket.import(this, "MyOtherBucket", props.theBucketRefProps);
+
+	    // Do something with myOtherBucket
+        }
+    }
+
+Finally, connect the dots.
+
+.. code-block:: ts
+
+    const app = new cdk.App(process.argv);
+
+    const myStack = new HelloCdkStack(app, "HelloCdkStack");
+    new MyCdkStack(app, "MyCdkStack", {
+        theBucketRefProps: myStack.myBucketRefProps
+    });
+
+    process.stdout.write(app.run());
+
+.. _using_cfn_template:
+
+Using an Existing |CFN| Template
+================================
+
+The |cdk| provides a mechanism that you can use to
+incorporate resources from an existing |CFN| template
+into your |cdk| app.
+For example, suppose you have a template,
+*my-template.json*,
+with the following resource,
+where **S3Bucket** is the logical ID of the bucket in your template:
+
+.. code-block:: json
+
+   "S3Bucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+          ...
+      }
+   }   
+   
+You can include this bucket in your |cdk| app,
+as shown in the following example
+(note that you cannot use this method in an |l2| construct):
+
+.. code-block:: ts
+
+   import { FnGetAtt } from '@aws-cdk/core';
+   import { readFileSync } from 'fs'
+   
+   new Include(this, "ExistingInfrastructure", {
```

Comment:
If you're going to show the imports, also include the import that brings in the `Include`.

And we don't really do unscoped imports like these anymore. These days, we encourage writing it like:

```ts
import cdk = require('@aws-cdk/cdk');

new cdk.Include(...);
```

Also, `@aws-cdk/core` library has changed its name.